specs/phase1/shard-transition.md

```diff
@@ -50,6 +50,9 @@ def verify_shard_block_message(beacon_state: BeaconState,
                                shard: Shard) -> bool:
     assert block.shard_parent_root == shard_state.latest_block_root
     assert block.slot == slot
+    next_slot = Slot(beacon_state.slot + 1)
```

Comment:
## ISSUE
We have two places use `get_shard_transition`
1. From validator guide `get_shard_transition`: `beacon_state` the lookahead state where slot = head_beacon_state + 1
2. From shark choice rule `on_shard_block`: `beacon_state` is the head block.

However, #1704 will remove (1). So I leave it here.

Reply:
Second thought: https://github.com/ethereum/eth2.0-specs/pull/1875